Libraries/Image/resolveAssetSource.js

```diff
@@ -58,22 +59,27 @@ function getOfflinePath() {
   return _offlinePath;
 }
 
+function getResourceIdentifier(asset) {
+  var assetDir = getBasePath(asset);
+  // E.g. 'assets_awesomemodule_icon'
+  // The Android resource system picks the correct scale.
+  return (assetDir + '/' + asset.name)
+    .toLowerCase()
```

Comment:
@zahanm, I have refactored the code. I put all the logic in a new file called `assetPathUtils.js` inside `local-cli`. Now, both `getAssetDestPathAndroid.js` and `resolveAssetSource.js` will reference this file and use the functions declared there, so there is no more logic duplication.